.env.test

```
@@ -1,6 +1,6 @@
 GOVUK_NOTIFY_API_KEY=override-locally
-CHECK_RECORDS_DOMAIN=http://127.0.0.1
-HOSTING_DOMAIN=http://127.0.0.1
+CHECK_RECORDS_DOMAIN=http://check_records.lvh.me
+HOSTING_DOMAIN=http://qualifications.lvh.me
```

Nice idea, however this introduces a dependency on a 3rd party service which hasn't been [entirely reliable](https://stackoverflow.com/a/67877376) in its recent history.

I've done some quick testing and it looks like we can use (for example):

```suggestion
CHECK_RECORDS_DOMAIN=check-records.localhost
HOSTING_DOMAIN=qualifications.localhost
```

With the corresponding tweaks to config along the lines of what you've done elsewhere in this PR, this should give us passing tests and a workable solution for different hostnames in dev.

---

Fine by me.